proxy-node-gateway/src/main/java/uk/gov/ida/notification/GatewayApplication.java

```diff
@@ -118,12 +118,6 @@ public void run(final GatewayConfiguration configuration,
                 environment,
                 "hub-metadata");
 
-        registerMetadataHealthCheck(
-                connectorMetadataResolverBundle.getMetadataResolver(),
```

Comment:
Does this mean we can remove the connectorMetadataResolverBundle that's instantiated on line 104?

Also, as the gateway isn't using the hub metadata in anyway, can we get rid of the hub metadata health check just above this one?

Reply:
There's actually quite a lot of stuff in the Application class which can be removed now I think, I'll get a PR up for it.